modules/bullet/collision_object_bullet.cpp

```diff
@@ -165,11 +165,20 @@ bool CollisionObjectBullet::has_collision_exception(const CollisionObjectBullet
 	return !bt_collision_object->checkCollideWith(p_otherCollisionObject->bt_collision_object);
 }
 
-void CollisionObjectBullet::prepare_object_for_dispatch() {
+void CollisionObjectBullet::reload_body() {
+	need_body_reload = true;
+}
+
+void CollisionObjectBullet::dispatch_callbacks() {}
+
+void CollisionObjectBullet::pre_process() {
 	if (need_body_reload) {
 		do_reload_body();
-		need_body_reload = false;
+	} else if (need_collision_fm_reload) {
+		do_reload_collision_fm();
 	}
```

Comment:
As above: `needs_collision_filters_reload` (and I'd put an "s" after "need") and `do_reload_collision_filters()`.

Reply:
Changed